content/en/docs/setup/install/operator/index.md

```diff
@@ -224,48 +224,63 @@ $ helm install istio-operator manifests/charts/istio-operator \
 
 {{< text bash >}}
-$ kubectl get pods -n istio-system -l app=istiod
-NAME                             READY   STATUS    RESTARTS   AGE
-istiod-5f4f9dd5fc-4xc8p          1/1     Running   0          10m
-istiod-1-8-1-55887f699c-t8bh8    1/1     Running   0          8m13s
+$ cat example-istiocontrolplane-1-8-1.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  namespace: istio-system
+  name: example-istiocontrolplane-1-8-1
+spec:
+  revision: 1-8-1
+  profile: default
 {{< /text >}}
 
+Apply the updated `IstioOperator` CR to the cluster, then you will have two control plane deployments and services running side-by-side:
+
 {{< text bash >}}
-$ kubectl -n istio-system get svc -l app=istiod
-NAME            TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)                                         AGE
-istiod          ClusterIP   10.87.7.69   <none>        15010/TCP,15012/TCP,443/TCP,15014/TCP,853/TCP   10m
-istiod-1-8-1    ClusterIP   10.87.4.92   <none>        15010/TCP,15012/TCP,443/TCP,15014/TCP,853/TCP   7m55s
+$ kubectl get pod -n istio-system -l app=istiod
+NAME                            READY   STATUS    RESTARTS   AGE
+istiod-1-8-1-597475f4f6-bgtcz   1/1     Running   0          64s
+istiod-6ffcc65b96-bxzv5         1/1     Running   0          2m11s
+{{< /text >}}
+
+{{< text bash >}}
+$ kubectl get svc -n istio-system -l app=istiod
+NAME           TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                                         AGE
+istiod         ClusterIP   10.104.129.150   <none>        15010/TCP,15012/TCP,443/TCP,15014/TCP,853/TCP   2m35s
+istiod-1-8-1   ClusterIP   10.111.17.49     <none>        15010/TCP,15012/TCP,443/TCP,15014/TCP           88s
 {{< /text >}}
 
 To complete the upgrade, label the workload namespaces with `istio.io/rev=1-8-1` and restart the workloads, as
 explained in the [Data plane upgrade](/docs/setup/upgrade/canary/#data-plane) documentation.
 
 ## Uninstall
 
-If you used the operator to perform a canary upgrade of the control plane, you can uninstall the old control plane and keep the new one by running the following command:
+If you used the operator to perform a canary upgrade of the control plane, you can uninstall the old control plane and keep the new one by deleting the old in-cluster `IstioOperator` CR, which will uninstall the old revision of Istio:
 
 {{< text bash >}}
-$ istioctl operator remove --revision <revision>
+$ kubectl delete istiooperators.install.istio.io -n istio-system example-istiocontrolplane
 {{< /text >}}
 
-Otherwise, delete the in-cluster `IstioOperator` CR, which will uninstall all revisions of Istio that may be running:
+Wait until Istio is uninstalled - this may take some time.
+
+Delete the specific revision of Istio operator by running the following command:
 
 {{< text bash >}}
-$ kubectl delete istiooperators.install.istio.io -n istio-system example-istiocontrolplane
+$ istioctl operator remove --revision <revision>
```

Comment:
As we are using `1-8-1` as an example of revision. I think following will be better?
```
$ istioctl operator remove --revision 1-8-1
```

Reply:
Normally, new revision of istio operator should not removed, we just tell the end users how to remove unused revision of istio operator(old one in upgrade, new one in rollback)